pkg/server/endpoints/entryfetcher.go

```diff
@@ -53,33 +51,24 @@ func (a *AuthorizedEntryFetcherWithFullCache) FetchAuthorizedEntries(ctx context
 
 // RunRebuildCacheTask starts a ticker which rebuilds the in-memory entry cache.
 func (a *AuthorizedEntryFetcherWithFullCache) RunRebuildCacheTask(ctx context.Context) error {
-	t := a.clk.Ticker(cacheReloadInterval)
-	defer t.Stop()
-
+	t := a.clk.Timer(cacheReloadInterval)
 	for {
 		select {
 		case <-ctx.Done():
 			a.log.Debug("Stopping in-memory entry cache hydrator")
 			return nil
 		case <-t.C:
-			start := time.Now()
-			cache, err := buildCache(ctx, a.metrics, a.ds)
-			end := time.Now()
-			hydrateLog := a.log.WithField(telemetry.ElapsedTime, end.Sub(start))
+			cache, err := a.buildCache(ctx)
 			if err != nil {
-				hydrateLog.WithError(err).Error("Failed to reload entry cache")
+				a.log.WithError(err).Error("Failed to reload entry cache")
 			} else {
-				hydrateLog.Debug("Reloaded entry cache")
+				a.log.Debug("Reloaded entry cache")
```

Comment:
I think we're missing the rebuild time from the log now?

Reply:
I intentionally removed it because it is extremely high cardinality and because this information is already emitted as a timer metric. It also makes it very difficult to test the contents of the log entries using `spiretest.AssertLogs()` because the value of the field is unpredictable.

Reply:
That's fair. The intention is to remove the log line anyway before we release. Considering the timing is available on the metric, seems ok to not worry about it here.